src/main/java/com/google/sps/servlets/SearchServlet.java

```diff
@@ -58,7 +58,7 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // List of all the tags we are searching for
     List<String> searchTags =
-        new ArrayList<String>(Arrays.asList(request.getParameterValues("tags")));
+        new ArrayList<String>(Arrays.asList(request.getParameter("tags").split(",")));
```

Comment:
If this was originally converted using JSON.stringify, are the brackets at the ends and the quotation marks taken care of?

Reply:
It was not converted using JSON.stringify, its a comma separated list as part of the query string (no brackets).